packages/expo-updates/ios/EXUpdates/UpdatesUtils.swift

```diff
@@ -133,14 +133,22 @@ public final class UpdatesUtils: NSObject {
     }
   }
 
-  internal static func isNativeDebuggingEnabled() -> Bool {
+  public static func isNativeDebuggingEnabled() -> Bool {
```

Comment:
```suggestion
  internal static func isNativeDebuggingEnabled() -> Bool {
```

keeping as internal?

Reply:
This needs to be public so that brownfield apps with custom initialization can access it.